src/app/organization/organization.service.ts

```diff
@@ -55,6 +55,14 @@ export class OrganizationService {
     return this.http.post('/staff', employee);
   }
 
+  /**
+   * @param {string} staffId Staff ID of employee.
```

Comment:
```suggestion
   * @param {string} employeeId Employee ID
```

Update the parameter.